draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
-# Discovery of associated resolvers {#first-usecase}
-
+{: #first-usecase}
+# Discovery of associated resolvers
 A client may begin with information from the attached networks, and/or resolvers known from configuration.
 This information may be used to then discover one or more associated or equivalent encrypted resolvers.
 These are expected to be operated by the same entity that provides the resolver the client started with, but
-may not have identical behaviour. For example one entity may offer a set of encrypted resolvers with different levels of
-filtering or proximity.
+may not have identical behaviour. For example one entity may offer a set of encrypted resolvers with different levels
+of filtering or proximity.
+
+These resolver upgrade mechanisms can either add new parameters to existing provisioning mechanisms
```

Comment:
I don't think this draft should discuss possible discovery mechanisms. It should only focus on use cases and requirements.

Reply:
This paragraph came from Tommy and Chris so I'll look to them to respond. Personally I think the second part " provide a way to communicate with" is ok, but I can see the point about "adding information to options in DHCP/..." which is unnecessarily specific.

Reply:
Works for me.

Reply:
I don't think this was intended to describe the mechanisms, but just give examples of what directly provisioning would look like. Could we just have the list in parentheses say "for example,"?

Reply:
Ok I've now added the "for example".